pkg/skaffold/build/local/jib_maven.go

```diff
@@ -62,14 +69,47 @@ func (b *Builder) buildJibMavenToRegistry(ctx context.Context, out io.Writer, wo
 
 // generateMavenArgs generates the arguments to Maven for building the project as an image called `skaffoldImage`.
 func generateMavenArgs(goal string, skaffoldImage string, a *latest.JibMavenArtifact) []string {
-	command := []string{"prepare-package", "jib:" + goal, "-Dimage=" + skaffoldImage}
+	var command []string
+	if a.Module == "" {
+		// single-module project
+		command = []string{"--non-recursive", "prepare-package", "jib:" + goal, "-Dimage=" + skaffoldImage}
```

Comment:
Someone was asking how to skip tests? Perhaps this can be in a followup, but maybe we should allow setting skipTests = false/true? or should that just be done in a profile?

Reply:
actually just ignore this, users should set everything up in the maven build directly.